tests/src/main/java/com/hazelcast/stabilizer/tests/map/helpers/IntegerStringEntryListener.java

```diff
@@ -67,31 +67,11 @@ public void mapCleared(MapEvent mapEvent) {
     }
 
     private void delay() {
-        if (maxDelayMs != 0) {
+        if (minDelayMs >= 0 && maxDelayMs > 0) {
```

Comment:
Nevertheless a good catch. A basic range check should be done in the constructor, not on each `delay()` call. The `maxDelayMs > 0` has to stay, but the check for negative values is now in the constructor (fail fast principle).